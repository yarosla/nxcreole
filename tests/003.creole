=This is //a test.
== Subheading ==

! center 1
!
! center 2

-hi
-bye

*hi bob!
**--hello
****how do you do
**
**bye
* bye -- bye

# 1
## 11
## 12
### 121
#### 1211

Links here: [[link1]], http://link2:8080/test?q=%aa%bb&b=123, [[link4|title]].

Images here: {{image1}}, {{image2|IMG
  2}}

-- Placeholder: <<<ABC>>>
----
---- not HR

{{{  **block**
   of {{{pre~}}}
}}}
{{{>>block
   of {{{pre}}}}}}
{{{Some inline {{**nowiki**}}}}}

>  This is blockquote.
>
> With blank line.