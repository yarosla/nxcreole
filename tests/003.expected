<h1>This is <em>a test.</em></h1>
<h2>Subheading</h2>
<div class="center">center 1</div>
<div class="center">&nbsp;</div>
<div class="center">center 2</div>
<ul><li>hi</li>
<li>bye</li></ul>
<ul><li>hi bob!<ul><li>--hello</li>
<li><strong>how do you do</strong></li>
<li>&nbsp;</li>
<li>bye</li></ul>
</li>
<li>bye â€“ bye</li></ul>
<ol><li>1<ol><li>11</li>
<li>12<ol><li>121<ol><li>1211</li></ol>
</li></ol>
</li></ol>
</li></ol>
<p>Links here: <a href="link1">link1</a>, <a href="http://link2:8080/test?q=%aa%bb&amp;b=123">http://link2:8080/test?q=%aa%bb&amp;b=123</a>, <a href="link4">title</a>.</p>
<p>Images here: <img src="image1" />, <img src="image2" alt="IMG
  2" /></p>
<p>-- Placeholder: &lt;&lt;&lt;Placeholder:ABC&gt;&gt;&gt;</p>

<hr/>
<p>---- not HR</p>
<pre>**block**
   of {{{pre}}}</pre>
<pre>&gt;&gt;block
   of {{{pre}}}</pre>
<p><span class="nowiki">Some inline {{**nowiki**}}</span></p>
<blockquote>This is blockquote.<br/><br/>
With blank line.</blockquote>
